core/primitives/src/views.rs

```diff
@@ -1016,3 +1016,17 @@ pub struct GasPriceView {
     #[serde(with = "u128_dec_format")]
     pub gas_price: Balance,
 }
+
+/// Different types of finality.
+#[derive(Serialize, Deserialize, BorshSerialize, BorshDeserialize, Clone, Debug, PartialEq, Eq)]
+pub enum Finality {
+    None,
+    DoomSlug,
+    NFG,
+}
+
+impl Default for Finality {
+    fn default() -> Self {
+        Finality::NFG
```

Comment:
This is also technically a breaking change, but since the new API has not reached master, we are good. Do you think it is what most RPC users want?

Reply:
Yes. I already changed the backward compatible part to use `Finality::None`. I think a lot of user will want `NFG` finality to be 100% sure.

Reply:
I would argue that the default should be Doomslug, which is faster and pretty safe.
NFG is a choice -> you want slower, for higher security.

Reply:
Disregard my above comment, quick survey shows people actually prefer NFG to be the default.